javascript/javascript2/week1/Week1HW.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+//Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const shortestWord = getShortestWord(danishWords); // returns 'ø'
+function getShortestWord(words) {
+  let newWord = danishWords[0];
+  for (let i = 0; i < danishWords.length; i++) {
+    if (danishWords[i].length <= newWord.length) {
+      newWord = danishWords[i];
+      danishWords[i] = danishWords[0];
+      danishWords[0] = newWord;
```

Comment:
why are you assigning the variable so weirdly here? Dont you only need `newWord = danishWords[i]` and later `return newWord`

Reply:
right! it is done.